content/pt-br/docs/concepts/workloads/pods/init-containers.md

```diff
@@ -0,0 +1,263 @@
+---
+title: contêineres de inicialização
+content_type: concept
+weight: 40
+---
+
+<!-- overview -->
+
+Essa página provê uma visão geral de contêineres de inicialização: especializada nos contêineres que executam antes de contêineres de aplicação em um {{< glossary_tooltip text="Pod" term_id="pod" >}}. Contêineres de inicialização podem conter utilidades ou configurações que não estão presentes em uma imagem de uma aplicação.
+
+Você pode especificar contêineres de inicialização em uma especificação de Pod ao longo de um array de `containers` (que descreve app contêineres).
+
+
+<!-- body -->
+
+## Entendendo contêineres de inicialização
+
+Um {{< glossary_tooltip text="Pod" term_id="pod" >}} pode ter múltiplos contêineres executando aplicações com ele, mas ele também pode ter um ou mais contêineres de inicialização, que são executados antes dos contêineres de aplicação serem iniciados.
+
+Contêineres de inicialização são exatamente como contêineres regulares, exceto:
+
+* Contêineres de inicialização executam sempre até completarem.
+* Cada contêiner de inicialização deve completar com sucesso antes de ir para o próximo início.
+
+Se o contêiner de inicialização do Pod falhar, o kubelet reinicia repetidamente aquele contêiner de inicialização
```

Comment:
Missing part of the sentence here (original reads "the kubelet restarts that init container until it succeeds"). Also, the original sentence reads "a init container", so the translation should be "um contêiner de inicialização".